include/rocksdb/db.h

```diff
@@ -520,6 +539,29 @@ class DB {
       values++;
     }
   }
+  virtual void MultiGet(const ReadOptions& options,
```

Comment:
Nit: Separate this function with the previous one with an empty line.

Reply:
Added empty line